docs/software/gaming.md

```diff
@@ -242,49 +242,53 @@ When ready, select Start to launch the emulator. Have fun!
 
 ### FAQ
 
-- How can I transfer Kickstarts & Floppy Images to the device?  
-  Make sure you have one of DietPi's File Servers installed.
+**How can I transfer Kickstarts & Floppy Images to the device?**  
```

Comment:
This needs to be a header to satisfy Markdownlint, and `####` anyway has the same visual result, so should be fine. Same for below cases.

Reply:
Nearly the same: Same font size but larger distance between the lines.  
Changed all the FAQ entries.